### PR TITLE
Fix UI/UX of number inputs in Firefox in plagiarism-inspector

### LINKED ISSUE #3485: Plagiarism Detection: Thresholds are not editable in FF
### Describe the bug

In Firefox (FF) you can't edit the Similarity Threshold and Minimum Score of the Plagiarism Detection in Artemis

#### To Reproduce

1. Use Firefox
2. Click on Check Plagiarism in the Instructor View of an Exercise and open the Plagiarism Detection
3. Try to edit the values

#### Expected behavior
Values shall be changed.

#### Screenshots

image

### Environment

<details><pre>
 - OS: Windows 10
 - Browser Firefox
 - Version 88.0.1 (64 bit)
</pre></details>

### PR DESCRIPTION
**Pull request by joschahenningsen**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).

### Motivation and Context
Plagiarism detection thresholds are not editable in Firefox (Fixes #3485)

### Description

Firefox doesn't do the best job at rendering number inputs with padding. Instead of doing some magic css I just removed the right padding on the inputs.

### Steps for Testing

0. Use Firefox
1. Log in to Artemis
2. Navigate to Plagiarism detection on a programming exercise
3. Test if inputs for thresholds are editable (by keyboard input and using the spinner buttons)

Please note that the weird inconsistent formatting changes have been applied by some pre commit hook and I'm lacking the energy to fight it. :smile:

### CODE REVIEW

File: src/main/webapp/app/exercises/shared/plagiarism/plagiarism-inspector/plagiarism-inspector.component.html [hunk 1]
```diff
@@ -57,7 +47,16 @@ <h4 class="plagiarism-page-title" (click)="showOptions = !showOptions" [class.ac
                     <div class="plagiarism-option-label" jhiTranslate="artemisApp.plagiarism.minimum-score">Minimum Score</div>
                 </label>
                 <fa-icon [icon]="'question-circle'" placement="bottom" [ngbTooltip]="'artemisApp.plagiarism.minimum-score-tooltip' | artemisTranslate" container="body"></fa-icon>
-                <input required type="number" [disabled]="!enableMinimumScore" class="form-control" min="0" max="100" id="plagiarism-minimum-score" [(ngModel)]="minimumScore" />
+                <input
+                    required
+                    type="number"
+                    [disabled]="!enableMinimumScore"
+                    class="form-control pr-0"
```
Comment (FrankeLukas) [inline src/main/webapp/app/exercises/shared/plagiarism/plagiarism-inspector/plagiarism-inspector.component.html:54]: Note: `pr-0` will need to be renamed if we merge this after #3470. If this is merged first #3470 needs to be adjusted.

**Review by FrankeLukas — APPROVED** [review]
Works.

File: src/main/webapp/app/exercises/shared/plagiarism/plagiarism-inspector/plagiarism-inspector.component.html (hunk 1 above)
Comment (joschahenningsen) [reply src/main/webapp/app/exercises/shared/plagiarism/plagiarism-inspector/plagiarism-inspector.component.html:54]: Thanks for the hint, I'll change it

**Review by NicolasUllmann — APPROVED** [review]
Tested on Server 1. Everything worked as expected.

**Review by willergeorg — APPROVED** [review]
Tested on TS1. Works for me as well.

**Pull request merged**